tests/test_setup.py

```diff
@@ -53,3 +53,10 @@ def test_celery_setup():
 
     result = add.delay(7, 3)
     assert result.wait() == 10
+
+
+@pytest.mark.django_db
```

Comment:
I actually just realized that you need to mark this one as transaction=True, otherwise the tests will fail (you can see this on the travis run associated with this pull request).

I believe this is because they call an external command, and the pytest runner is not able to know what happened there so it can roll things back. In any case changing this to `@pytest.mark.django_db(transaction=True)` seems to fix this problem. You should verify this yourself locally by running the tests:

http://treeherder.readthedocs.io/common_tasks.html#running-the-tests

(you only need to run the python tests)

Sorry for not noticing this problem earlier